src/pages/FlowRun/Actions.vue

```diff
@@ -233,7 +234,7 @@ export default {
         </div>
       </template>
       <span v-if="isReadOnlyUser">
-        Read-only users cannot delete flows
+        You don't have permission to delete flows
```

Comment:
This will need delete:flow permissions?

Reply:
I was thinking delete:run since it's trying to delete a flow run?